src/transaction/locator_sr.c

```diff
@@ -13937,6 +13938,162 @@ has_errors_filtered_for_insert (std::vector<int> error_filter_array)
 }
 // *INDENT-ON*
 
+/* locator_put_classname_entry () - Put a new entry in the locator_Mht_classnames, used in PTS replicator (atomic_replicator)
+ *
+ * return	     : void
+ * thread_p (in)     : thread entry
+ * classname (in)    : classname to be inserted
+ * classoid (in)     : class oid of the classname
+ */
+void
+locator_put_classname_entry (THREAD_ENTRY * thread_p, const char *classname, const OID * classoid)
+{
+  assert (classname != NULL);
```

Comment:
Nit: use `nullptr` for all new code.
```suggestion
  assert (classname != nullptr);
```